sources/Inputs/include/Behaviours.h

```diff
@@ -122,5 +122,52 @@ struct ConverterInterface {
   boost::optional<bool> voltageRegulationOn;  ///< voltage regulation parameter, for VSC converters only
   std::shared_ptr<HvdcLine> hvdcLine;         ///< hvdc line this converter is contained into
 };
+
+/// @brief Static var compensator (SVarC) behaviour
+struct StaticVarCompensator {
+  using SVarCid = std::string;  ///< alias for static var compensator id
+
+  /**
+   * @brief Constructor
+   *
+   * @param id the id of the SVarC
+   * @param bMin the minimum b of the SVarC
+   * @param bMax the maximum b of the SVarC
+   * @param voltageSetPoint the voltage set point of the SVarC
+   * @param VNom the nominal voltage of the bus of the SVarC
+   * @param UMinActivation the low voltage threshold of the SVarC
```

Comment:
the low voltage activation threshold of SVarC
similar below

Reply:
OK